tests/wpt/web-platform-tests/html/semantics/embedded-content/media-elements/playing-the-media-resource/play-in-detached-document.html

```diff
@@ -12,12 +12,17 @@
   var v = doc.createElement("video");
   doc.body.appendChild(v);
   v.src = getVideoURI("/media/movie_5");
-  v.play();
+  var play_promise = v.play();
```

Comment:
> and its related spec is the step 2 of play() in https://html.spec.whatwg.org/multipage/media.html#dom-media-play

This was a wrong investigation; the root cause of `NotSupported` is thrown from `resource_selection_algorithm_sync`

https://github.com/servo/servo/blob/e4657c14961db58bedd0a13ba72667bcb470f34d/components/script/dom/htmlmediaelement.rs#L619-L626

and it failed to parse the URL so it runs the `dedicated media source failure steps`

https://github.com/servo/servo/blob/e4657c14961db58bedd0a13ba72667bcb470f34d/components/script/dom/htmlmediaelement.rs#L747-L754

But, I can pass the test case intermittently in my local env (both `TIMEOUT` for testharness and the test case) without changing the test case.

btw, I'm a little confused about this test case. If we cannot `play` in a `detached  document`, will the `timeupdate` be executed? Should we just check the `play()` promise be rejected instead?

cc @jdm @ferjm